README.md

```diff
@@ -299,12 +303,12 @@ containers.
 
 Note: We have to be careful when comparing memory consumption of JVM services, because the JVM pre-allocates memory it
 does not actually use. Therefore, we are not measuring the memory allocated by the JVM, but the memory consumed by the
-whole container, which turned out to be a more meaningful metric.
+whole container, which turned out to be a more meaningful metric. <mark>why is the container memory a better metric?</mark>
```

Comment:
In my tests the metrics about the JVM memory consumption were fluctuating very strong and I was not really able to recognize any patterns. 
The metrics of the containers were much more consistent and easier to decypher.

Reply:
Do you think that an extension to this note is helpful?